extension-requests/style.css

```diff
@@ -773,8 +773,43 @@ body {
   align-items: center;
   justify-content: space-between;
   position: relative;
-  height: 1.6rem;
+  height: 2rem;
+}
+
+.title-input-wrapper {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
+.title-input-error,
+.reason-input-error,
+.extension-input-error {
+  font-size: 12px;
+  color: var(--red-color);
 }
+
+.extension-request-update-toast {
+  position: fixed;
+  bottom: 20px;
+  right: 20px;
+  padding: 15px 20px;
```

Comment:
Can we try the following 8/4 point system for keeping things consistent
https://www.designsystems.com/space-grids-and-layouts/

Reply:
Done